src/js/tabs.js

```diff
@@ -0,0 +1,73 @@
+/*jshint esversion: 6 */
+
+// Create the tabs and append them to content div
+function buildTabs(tabs,tabsNames){
+  let tabsList = document.createElement('ul');
+  tabsList.setAttribute('class','tab-list');
+
+  for(let i = 0; i < tabsNames.length; i++){
```

Comment:
What do you think about using an array method such as `forEach` or `for of` instead of the `for` loop? :)